todolist.js

```diff
@@ -43,6 +43,17 @@ function main() {
 }
 
 
+function deleteTodo() {
+    let todoToDelete = prompt('What todo would you like to delete?');
+    for(let i = 0; 1 < todos.length; i++) {
```

Comment:
should be `i < todos.length`, not `1 < todos.length`

Reply:
fixed it!